huaweicloud/services/lts/resource_huaweicloud_lts_stream.go

```diff
@@ -0,0 +1,142 @@
+package lts
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/openstack/lts/huawei/logstreams"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+)
+
+func ResourceLTSStream() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceStreamCreate,
+		ReadContext:   resourceStreamRead,
+		DeleteContext: resourceStreamDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
```

Comment:
this may not work when the script is like this:
```hcl
resource "huaweicloud_lts_stream" "test_stream" {
}
```
pleae recheck it

Reply:
fixed